docs/docs/migrate/keptnapp/index.md

```diff
@@ -0,0 +1,63 @@
+# Migrating KeptnApp to KeptnAppContext
+
+The introduction of the `KeptnAppContext` resource
+in Keptn v0.11.0
+requires modifications to existing
+[KeptnApp](../../reference/crd-reference/app.md)
+resources that were generated manually.
+This page gives instructions for doing that.
+
+## Migration steps
+
+> **Warning**
+Migration is only required for
```

Comment:
I would prefer to have the admonition before the header.  If you disagree, just resolve this comment.